_test/general.test.php

```diff
@@ -5,13 +5,73 @@
  * @group plugin_findologicxmlexport
  * @group plugins
  */
-class general_plugin_findologicxmlexport_test extends DokuWikiTest {
+
+require_once(__DIR__ . '/../DokuwikiXMLExport.php');
+require_once(__DIR__ . '/../admin.php');
+
+class general_plugin_findologicxmlexport_test extends DokuWikiTest
+{
+    /**
+     * A page needs to have content. This is the placeholder for this value.
+     */
+    const PAGE_CONTENT_PLACEHOLDER = 'This page is for test purposes.';
+
+    /**
+     * Helper function to save a page.
+     *
+     * @param array /string $ids one or multible page ids
```

Comment:
Such dynamic typing (`array / string`) is rarely a good idea as it makes the code a lot more complex and your API less clear. Decide on one input type and go with that. In this case I guess that would be `array`.

Reply:
Yes you are right here. Thanks for that!